### DISCUSSION TITLE
Newsletter?

### DISCUSSION THREAD

**Discussion by ericwr** [livewire/flux#862] [Feature Requests]
Hi everyone,
First of all, thank you for this great initiative! I can't wait to be able to work with these components (I think I'll make the switch when there's a date picker available :)
To go further, here are a few components that would be great to have: date/week/month/period picker, timeline/calendar, enhanced datatable...
I've seen discussions already mentioning these components, but what's sorely missing (or maybe I've missed something) is the possibility of subscribing to a newsletter to get the latest news on this project!

Thanks in advance and I wish this project all the best!

**Comment by jeffchown** [comment]
@erwal45 You can view the main update information here: https://fluxui.dev/blog and GitHub technical release details here: https://github.com/livewire/flux/releases

**Reply by joshhanley** [reply]
@erwal45 Jeff's correct on how to see all of the regular updates. I do know Caleb sends some emails with updates too. Not sure if they just go to existing customers or if there is a way to sign up for them. Have asked the question and will let you know. 🙂
